### DISCUSSION TITLE
How to use local instalation of Bravura Font?

### DISCUSSION THREAD

**Discussion by charlesneimog** [DigiScore/neoscore#85] [Q&A] [answered]
In neoscore packages we have `_BRAVURA_PATH` and `_BRAVURA_METADATA_PATH`, I want to modify them to point to my local copy of Bravura.otf and bravura_metadata.json. There is some way that I can do this?

Thanks!

**Comment by charlesneimog** [comment, marked as answer]
I didn't understand that Bravura comes with neosore.

**Reply by ajyoon** [reply]
That's right. Also, we do ostensibly support other SMuFL-compliant music fonts, which can be loaded from a given file path with [`MusicFont`](https://neoscore.org/api/neoscore.core.music_font.html), though [it's not very well-tested](https://github.com/DigiScore/neoscore/issues/29).

**Reply by ajyoon** [reply]
actually, music fonts need to be registered ahead of time with [`neoscore.register_music_font`](https://neoscore.org/api/neoscore.core.neoscore.html#neoscore.core.neoscore.register_music_font), then they can be used in `MusicFont` by referencing the family name.
